modules/recording/recording/dashboard.tsx

```diff
@@ -23,6 +26,26 @@ const Dashboard = () => {
   const audioRecorderRef = useRef<AudioRecorder | null>(null);
   const analyserRef = useRef<AnalyserNode | null>(null);
   const animationFrameRef = useRef<number | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasTemplates, setHasTemplates] = useState(true);
+
+  useEffect(() => {
+    void fetchTemplates();
+  }, []);
+
+  const fetchTemplates = async () => {
+    setIsLoading(true);
+    const response = await templateService.getTemplates(1);
+    if (!response) return;
```

Comment:
If no response, setLoading is true forever.

Negative condition first, and reduce the if-else